web/pdf_page_view.js

```diff
@@ -982,12 +982,12 @@ class PDFPageView {
         showCanvas?.(true);
         await this.#finishRenderTask(renderTask);
 
-        this.#renderTextLayer();
-
         if (this.annotationLayer) {
           await this.#renderAnnotationLayer();
         }
 
+        this.#renderTextLayer();
+
```

Comment:
I *really* don't think that we should change the order of the text/annotation-layer rendering like this, since we've always rendered the textLayer first!

If the StructTree-data returned by the API, internally, depends on the order in which the text/annotation-data is being accessed then that really *must* be fixed in the worker-thread instead.